sirepo/pkcli/job_cmd.py

```diff
@@ -238,6 +238,13 @@ def _do_sequential_result(msg, template):
     return r
 
 
+def _do_simulation_db(msg, template):
+    try:
+        return template_common.simulation_db_dispatch(msg.data)
```

Comment:
This looks like a pattern to encapsulate. Perhaps these requests are actually one API with a method attached with attributes describing what level of "compute" they require.

Reply:
43797f40e . Thoughts? I'm shying away from using one API because I think that leads us back down the road of IOCTL.